setup.py

```diff
@@ -3,8 +3,21 @@
 from setuptools import setup, find_packages
 from pip._internal.req import parse_requirements
 
-install_reqs = parse_requirements('requirements.txt', session=False)
-test_reqs = parse_requirements('requirements_test.txt', session=False)
+parsed_install_reqs = parse_requirements('requirements.txt', session=False)
+install_reqs = []
+for ir in parsed_install_reqs:
```

Comment:
generally don't love these sorts of variable names but since we're truly doing it live it'll do

Reply:
agreed I hate all of it